com.vbam.visualboyadvancem.yaml

```
@@ -0,0 +1,119 @@
+app-id: com.vbam.visualboyadvancem
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

```suggestion
runtime-version: '23.08'
```

it's the current version